core/src/main/scala/cromwell/util/JsonEditor.scala

```diff
@@ -83,7 +83,17 @@ object JsonEditor {
 
   def outputs(json: Json): Json = includeJson(json, NonEmptyList.of("outputs")) |> (excludeJson(_, NonEmptyList.one("calls")))
 
-  def logs(json: Json): Json = includeJson(json, NonEmptyList.of("stdout", "stderr", "backendLogs"))
+  def logs(json: Json): ErrorOr[Json] = {
+    // Deletes all subworkflows found in the input by always returning `None`.
+    def deleteSubworkflows(o: JsonObject, j: Json): Option[ErrorOr[Json]] = None
+    for {
+      wf <- updateWorkflowCallsJson(json, deleteSubworkflows)
```

Comment:
Could you inline this as `_ => None`?

Reply:
Also... this seems like it would delete everything (ie not just subworkflows)?

Reply:
no this is fancy and only gets invoked for subworkflows

Reply:
`_ => None` ❌ 
`(_, _) => None` ✅